demo/index.html

```diff
@@ -141,7 +141,7 @@ <h1 class="c-main__title">React Components</h1>
               <a class="u-fg-inherit" href="tables">Tables</a>
             </div>
             <div class="u-mb-sm">
-              <a class="u-fg-inherit" href="tabs">Tabs</a>
+              <a class="u-fg-inherit" href="storybook">Tabs</a>
```

Comment:
For now, it would be good if this got us to `storybook/?path=/story/components-tabs--default`. Eventually, we'll get rid of this index page and replace it with storybook.

Reply:
This now deep-links to the Tabs example.